src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int height;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int height, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg + secondLeg) / 2 * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: " + String.format("%.1f", getArea())
+                + " sq. units, first leg: " + firstLeg + " units, second leg: " + secondLeg
+                + " units, height: " + height + " units, color: "
+                + getColor().toString().toLowerCase());
```

Comment:
The 'toString().toLowerCase()' method call is unnecessary here because enum names are constants and should be in English by convention. You can simply use getColor().name() to get the string representation of the enum constant.